index.html

```diff
@@ -21,145 +22,240 @@
 
 <body>
 	<div class="firstSection">
-		<header>
+		<header class="headerFirstSection">
 			<ul class="toolbar">
-				<li><a href="#">Diego Cuenca</a></li>
-				<li>
+				<li class="toolbarMobile"><a href="#">Diego Cuenca</a></li>
+				<li class="toolbarDesktop pushright">Portfolio</li>
+				<li class="toolbarDesktop">About</li>
+				<li class="toolbarDesktop">Contact</li>
+				<li class="toolbarDesktop">
+					<nav><img src="./icons/desktop/mail.png" alt="mailIcon"></nav>
+				</li>
+				<li class="toolbarMobile mobileIconmenu">
 					<nav><img src="./icons/iconMenu.png" alt="iconMenu"></nav>
 				</li>
 			</ul>
 		</header>
-		<section class="headline">
-			<h1>Hey There. I’m Diego</h1>
-			<h2>I’am a Software Developer</h2>
-			<p>I can help you build a product , feature or website Look through some of my work and experience! If you
-				like
-				what you see and have a project you need coded,
-				don’t hestiate to contact me.
-			</p>
-		</section>
-		<footer>
-			<ul class="media">
-				<li><img src="./icons/Git.png" alt="Github Icon"></li>
-				<li><img src="./icons/Linkedin.png" alt="LinkedIn Icon"></li>
-				<li><img src="./icons/D2.png" alt="D2 Icon"></li>
-				<li><img src="./icons/Twitter.png" alt="Twitter Icon"></li>
-				<li><img src="./icons/Microverse.png" alt="Microverse Icon"></li>
-			</ul>
-		</footer>
+		<div class="leftMedia">
+			<section class="headline">
+				<h1>Hey There. I’m Diego</h1>
+				<h2>I’am a Software Developer</h2>
+				<p>I can help you build a product , feature or website Look through some of my work and experience! If
+					you like what you see and have a project you need coded,
+					don’t hestiate to contact me.
+				</p>
+			</section>
```

Comment:
- I think it will be better to add a `</br>` tag just after `Hey There.` so that the next statement goes over to the next line. 
something like this 👇🏽 
```
<h1>Hey There. </br> I’m Diego</h1>
```
- I think you can add a little bit more margin-top to this entire `<section class="headline">` as well' 🤗 
- Currently, I think the font-size of your social icons are big. Can you reduce them a bit so that it looks like what we have on figma?? 🌸 

Here is what I can see, compared to what's on figma 👇🏽 
## Your design
![dieq](https://user-images.githubusercontent.com/99246359/200028792-0723dc8d-d962-413d-b006-4d8c6aac6dd2.PNG)

### Figma design

![dieqfig](https://user-images.githubusercontent.com/99246359/200028966-258f44ff-a834-4942-94ab-ed87b66cf2d5.PNG)

Reply:
Thanks for the improvements in my portfolio, I did it like your suggestions.